src/pipecat/services/tavus.py

```diff
@@ -0,0 +1,138 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+"""This module implements Tavus as a sink transport layer"""
+
+# import asyncio
+# from typing import Any
+import base64
+import requests
+import time
+
+# from enum import Enum
+# from typing import AsyncGenerator
+
+# import numpy as np
+
+from pipecat.frames.frames import ErrorFrame, Frame, TranscriptionFrame, TTSAudioRawFrame, TransportMessageUrgentFrame, TTSStartedFrame, TTSStoppedFrame
+from pipecat.processors.frame_processor import FrameDirection, FrameProcessor
+# from pipecat.services.ai_services import AIService
+from pipecat.transports.base_output import BaseOutputTransport
+from pipecat.transports.services.daily import DailyTransportClient, DailyParams
+# from pipecat.utils.time import time_now_iso8601
+
+from loguru import logger
+
+MIN_AUDIO_BUFFER_SIZE = int(16000 * 2 * 0.5) # 0.5 seconds
+
+class TavusVideoService(BaseOutputTransport):
```

Comment:
I think this should inherit from `FrameProcessor`  and just create `TransportMessageFrame`s. Those would be sent by the `transport.output()`

Reply:
`TransportMessageFrame` instead of `TransportMessageUrgentFrame`?

Reply:
No, sorry, `TransportMessageUrgentFrame` is good. I was just talking about a transport message in general. Whether it's urgent or not depends on your processor, which should actually be urgent in your case.